src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.Simple.cs

```diff
@@ -8,126 +8,193 @@ namespace System.Formats.Cbor
 {
     public partial class CborReader
     {
+        private const int SizeOfHalf = 2; // the size in bytes of an IEEE 754 Half-Precision float
+
+        /// <summary>
+        ///   Reads the next data item as a single-precision floating point number (major type 7)
+        /// </summary>
+        /// <returns>The decoded <see cref="float"/> value.</returns>
+        /// <exception cref="InvalidOperationException">
+        ///   the next data item does not have the correct major type --OR--
+        ///   the next simple value is not a floating-point number encoding --OR--
+        ///   the encoded value is a double-precision float
+        /// </exception>
+        /// <exception cref="FormatException">
+        ///   invalid CBOR encoding data --OR--
+        ///   unexpected end of CBOR encoding data --OR--
+        ///   CBOR encoding not accepted under the current conformance level
```

Comment:
For all of these FormatExceptions, each clause should be a valid English sentence when preceded by "FormatException is thrown if " (https://github.com/dotnet/dotnet-api-docs/wiki/Exceptions).  So, probably something like

```suggestion
        ///   the next value has an invalid CBOR encoding.
        ///
        ///   the next value uses a CBOR encoding that is not valid under the current conformance level.
```

(You can keep including the one about unexpectedly running out of data, but in a similar form.  I'd just qualify that as "invalid", personally, but maybe there's a semantic difference worth calling out.)